src/oop/app/src/Scrapper.php

```diff
@@ -7,3 +7,30 @@
  * Note: Dont forget to create variables for TransportInterface and ParserInterface objects.
  * Note: Also you can add your methods if needed.
  */
+
+namespace src\oop\app\src;
+
+use src\oop\app\src\Models\Movie;
+use src\oop\app\src\Models\MovieInterface;
+use src\oop\app\src\Parsers\ParserInterface;
+use src\oop\app\src\Transporters\TransportInterface;
+
+class Scrapper
+{
+    private TransportInterface $transport;
+
+    private ParserInterface $parser;
+
+    public function __construct(TransportInterface $transport, ParserInterface $parser)
```

Comment:
Please change it to
`public function __construct(private TransportInterface $transport, private ParserInterface $parser)`